docs/reference/docs/get.asciidoc

```diff
@@ -228,5 +228,7 @@ it's current version is equal to the specified one. This behavior is the same
 for all version types with the exception of version type `FORCE` which always
 retrieves the document.
 
-Note that Elasticsearch do not store older versions of documents. Only the current version can be retrieved.
+Note that even though Elasticsearch does not delete versions of documents in
+an index to ensure constant sequential writes, older versions of documents are
+unreachable. Only the current version can be retrieved.
```

Comment:
Sorry about that. I think the better action to take is to pull in wording that already exists in the definitive guide found here - https://github.com/elastic/elasticsearch-definitive-guide/blob/master/030_Data/25_Update.asciidoc

"Internally, Elasticsearch has marked the old document as deleted and added an entirely new document. The old version of the document doesn’t disappear immediately, although you won’t be able to access it. Elasticsearch cleans up deleted documents in the background as you continue to index more data."

If you feel this is more appropriate I will update to include that which should be less ambiguous and also consistent with other documentation/wording.